Formula/k/kubectl-rook-ceph.rb

```diff
@@ -0,0 +1,25 @@
+class KubectlRookCeph < Formula
+  desc "Rook plugin for Ceph management"
+  homepage "https://rook.io/"
+  url "https://github.com/rook/kubectl-rook-ceph.git",
+      tag:      "v0.8.0",
+      revision: "aba01b0bd31c0efd4ac772c4a26ced7e6f2fff81"
+  license "Apache-2.0"
+  head "https://github.com/rook/kubectl-rook-ceph.git", branch: "master"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w", output: bin/"kubectl-rook_ceph"), "cmd/main.go"
+  end
+
+  test do
+    assert_match "kubectl rook-ceph provides common management and troubleshooting tools for Ceph.",
+                 shell_output("#{bin}/kubectl-rook_ceph --help")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
I was mostly using the [k9s Formular](https://github.com/Homebrew/homebrew-core/blob/master/Formula/k/k9s.rb#L39) as source which also does just this little check. 

I don't think we can do any more advanced tests, since all other commands require a fully running kubernetes cluster with rook running inside.

I could add a negative test that would error and say that a cluster is required, but I'm not sure if that's still useful.

Reply:
yeah, we should have a better test for that formula.

Reply:
Added negative test